# Define the goal
string move_command
--- 
# Define the result
bool movement_finished
---
# Define a feedback message
geometry_msgs/Point current_position
